.remarkrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   plugins: [
-    './packages/remark-mdx',
+    'remark-frontmatter',
```

Comment:
won't remark-mdx still be needed to parse MDX files?

Reply:
`eslint-plugin-mdx` enables `remark-mdx@1` automatically. (Because of gastby, we are still using mdx@1).

See also https://github.com/mdx-js/mdx/pull/1490#discussion_r600558337